### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by nathan-fiscaletti** [evanoberholster/timezoneLookup#15]
This doesn't seem to function when run on Windows.

```
# github.com/evanoberholster/timezoneLookup/v2
C:\Users\Nathan Fiscaletti\go\pkg\mod\github.com\evanoberholster\timezone!lookup\v2@v2.0.0\timezone.go:222:17: undefined: syscall.Mmap
C:\Users\Nathan Fiscaletti\go\pkg\mod\github.com\evanoberholster\timezone!lookup\v2@v2.0.0\timezone.go:222:64: undefined: syscall.PROT_READ
C:\Users\Nathan Fiscaletti\go\pkg\mod\github.com\evanoberholster\timezone!lookup\v2@v2.0.0\timezone.go:222:80: undefined: unix.MAP_SHARED
C:\Users\Nathan Fiscaletti\go\pkg\mod\github.com\evanoberholster\timezone!lookup\v2@v2.0.0\timezone.go:227:17: undefined: syscall.Munmap
```

Any plans to add windows support?

**Comment by christophergm** [comment]
This issue would be resolved by [PR-16](https://github.com/evanoberholster/timezoneLookup/pull/16) with no performance impact on non-windows.  

@evanoberholster this is a nice package and cross-platform support would make adoption of v2 easier.  Are you taking PRs still?

**Comment by nathan-fiscaletti** [comment]
Resolved in #16
